collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -202,6 +208,9 @@ public View getView(int position, View convertView, ViewGroup parent) {
         mDrawerList.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                Log.e("selected ",mSelectedSortingOrder+"");
+                parent.getChildAt(mSelectedSortingOrder).setBackgroundColor(Color.parseColor("#DDDDDD"));
+                view.setBackgroundColor(Color.parseColor("#2196F3"));
```

Comment:
Please move this value to [colors.xml](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/res/values/colors.xml)